system/gluster_volume.py

```diff
@@ -91,7 +96,7 @@
 
 EXAMPLES = """
 - name: create gluster volume
-  gluster_volume: state=present name=test1 brick=/bricks/brick1/g1 rebalance=yes hosts:"{{ play_hosts }}"
+  gluster_volume: state=present name=test1 brick=/bricks/brick1/g1 rebalance=yes cluster:"{{ play_hosts }}"
```

Comment:
No, it was renamed. It was named `hosts` in previous version of this module, names `glusterfs` (it's in stable-1.9 branch now)